troughs_and_waves/troughs_and_waves_plugin.py

```diff
@@ -0,0 +1,51 @@
+#  (C) Copyright 2010-2020 Enthought, Inc., Austin, TX
+#  All rights reserved.
+
+from force_bdss.api import plugin_id
+from force_bdss.core_plugins.service_offer_plugin import \
+    ServiceOfferExtensionPlugin
+
+from .gaussian.gaussian_data_source_factory import GaussianDataSourceFactory
+
+
+PLUGIN_VERSION = 0
+
+
+class TroughsAndWavesPlugin(ServiceOfferExtensionPlugin):
```

Comment:
At this stage we can just inherit from the `force_bdss.api.BaseExtensionPlugin` class here. If we add any `ContributedUI` classes later on we can easily switch to the `ServiceOfferExtensionPlugin` instead.

It also means you don't have to implement the empty `get_contributed_uis`, `get_data_views` and `get_service_offer_factories` methods